src/controller/python/test/test_scripts/mobile-device-test.py

```diff
@@ -95,35 +76,53 @@ def main():
     #
     # Issue: #15688
     #
-    # asyncio.run(test.TestMultiFabric(ip=options.deviceAddress,
+    # asyncio.run(test.TestMultiFabric(ip=address.decode("utf-8"),
     #                                  setuppin=20202021,
     #                                  nodeid=1))
 
     logger.info("Testing closing sessions")
-    FailIfNot(test.TestCloseSession(nodeid=1), "Failed to close sessions")
-
-    logger.info("Testing resolve")
-    FailIfNot(test.TestResolve(nodeid=1),
-              "Failed to resolve nodeid")
-
-    # Still test network commissioning
-    logger.info("Testing network commissioning")
-    FailIfNot(asyncio.run(NetworkCommissioningTests(devCtrl=test.devCtrl, nodeid=1).run()),
-              "Failed to finish network commissioning")
-
+    FailIfNot(test.TestCloseSession(nodeid=device_nodeid),
+              "Failed to close sessions")
+
+
+@click.command()
+@click.option("--controller-nodeid", default=TEST_CONTROLLER_NODE_ID, type=int, help="NodeId of the controller.")
+@click.option("--device-nodeid", default=TEST_DEVICE_NODE_ID, type=int, help="NodeId of the device.")
+@click.option("--timeout", "-t", default=240, type=int, help="The program will return with timeout after specified seconds.")
+@click.option("--discriminator", default=TEST_DISCRIMINATOR, type=int, help="Discriminator of the device.")
+@click.option("--setup-pin", default=TEST_SETUPPIN, type=int, help="Setup pincode of the device.")
+@click.option("--commissioning", default='eth', type=click.Choice(['eth'], case_sensitive=False), help="The network type of the device, used during commissioning test.")
+@click.option('--enable-test', default=['all'], multiple=True, help='The tests to be executed.')
+@click.option('--disable-test', default=[], multiple=True, help='The tests to be excluded.')
+@click.option('--log-level', default='WARN', type=click.Choice(['ERROR', 'WARN', 'INFO', 'DEBUG']), help="The log level of the test.")
+def main(controller_nodeid, device_nodeid, timeout, discriminator, setup_pin, commissioning, enable_test, disable_test, log_level):
+    logger.info("Test Parameters:")
+    logger.info(f"\tController NodeId: {controller_nodeid}")
+    logger.info(f"\tDevice NodeId:     {device_nodeid}")
+    logger.info(f"\tTest Timeout:      {timeout}s")
+    logger.info(f"\tCommission Over:   {commissioning}")
+    logger.info(f"\tDiscriminator:     {discriminator}")
+    logger.info(f"\tEnabled Tests:     {enable_test}")
```

Comment:
Could you add more examples/comments regarding which tests can be enabled?

Reply:
Added to matter-repl.md